### PR TITLE
Update filterSNVs.sh

### PR DESCRIPTION
**Pull request by nickhsmith**
previously if there were multiple alleles in the vcf A > C,G it would throw an error because ASEReadCounter can't handle multiple alleles in a position. If the vcf was a multi-sample VCF this could cause issues.

The new functionality is to split A > C and A>G and then select only the heterozygous variants and header lines (only relevant ones for MAE anyway).

This way if sample A has heterozygous A>C and sample B has heterozygous A>G 

then for sample A:
split A>C,G into
A >C 0|1
A >G 0|0
the SelectVariants removes the alleles that aren't relevant for this sample leaving only
A >C 0|1

and for sample B
A >C 0|0
A >G 1|0
the SelectVariants removes the alleles that aren't relevant for this sample leaving only
A >G 1|0

### CODE REVIEW

**Review by vyepez88 — COMMENTED** [review]
this might not work with MT / chrM. Why not setting the seqlevelsStyle of the gtf file?

File: drop/modules/mae-pipeline/MAE/filterSNVs.sh [hunk 1]
```diff
@@ -22,13 +22,25 @@ samtools=$8
 tmp=$(mktemp)
 
 echo 'Filter SNVs'
+
+# if not doing QC
```
Comment (vyepez88) [inline drop/modules/mae-pipeline/MAE/filterSNVs.sh]: instead of 'if not doing QC', explain what is about to happen, or why is it needed

**Review by vyepez88 — COMMENTED** [review]
not sure why we need the new line 45 regarding the chr style, we already have a check on that afterwards

**Comment by nickhsmith** [comment]
> not sure why we need the new line 45 regarding the chr style, we already have a check on that afterwards

I believe that was because of the gene_name/mapping comparison. But I don't remember now.

**Review by vyepez88 — DISMISSED** [review]

**Pull request merged**